src/client/pages/OfferNew/Checkout/index.tsx

```diff
@@ -170,7 +170,7 @@ export type CheckoutProps = {
 export const Checkout = ({
   quoteBundleVariants,
   selectedQuoteBundleVariant,
-  onAddQuotes,
+  onUpsellAccepted,
```

Comment:
Now we are calling the same thing "deal" and "upsell" -- not crazy about that but not a deal breaker